configs/components/rubygem-concurrent-ruby.rb

```diff
@@ -1,6 +1,6 @@
 component 'rubygem-concurrent-ruby' do |pkg, settings, platform|
-  pkg.version '1.1.5'
-  pkg.md5sum '4409c2d6925d8448cb34a947eacaa29b'
+  pkg.version '1.1.6'
```

Comment:
Please do not take 1.1.6 https://github.com/ruby-concurrency/concurrent-ruby/pull/856#issuecomment-604124810 will be needed. 1.1.5 still works. (this is for ace-server)